package.json

```diff
@@ -73,6 +73,7 @@
     "rollup-plugin-node-globals": "^1.1.0",
     "rollup-plugin-node-resolve": "^3.0.0",
     "slash": "^1.0.0",
+    "source-map-support": "^0.5.0",
```

Comment:
Why is this here? Shouldn't it be a dependency of `jest-jasmine2`?

Reply:
You're right. I just moved it there